### ISSUE TITLE
content_script not updating

### ISSUE THREAD

**Issue by riccardoferretti** [chibat/chrome-extension-typescript-starter#18]
Once I load the extension in chrome, changes to `content_script.ts` don't seem to live reload.
A few notes:
- the generated file `dist/js/content_script.js` is updated
- changes to `background.ts` are propagated if I refresh the devtools for the background "page"
- changes to the popup are propagated if I close and reopen the popup

I have tried:
- reloading the page
- opening/closing the popup
- clicking "update" in the extensions panel

None of that worked, the only thing that worked was removing and re-adding the extension.

Any idea of why I have having this issue?
Thanks!

**Comment by riccardoferretti** [comment]
I actually just found a little reload icon from the extensions list page, which did the trick. 

Still wondering, is this the way to do it? Is there a way to trigger the refresh automatically?

**Comment by CodeDem** [comment]
@riccardoferretti , yes that's how we're supposed to do. Chrome plugin once built is not automatically reloaded.
